libunwind/src/libunwind.cpp

```diff
@@ -12,6 +12,7 @@
 #include <libunwind.h>
 
 #include "config.h"
+#ifndef __wasm__
```

Comment:
It seems like we simply don't want this file to build under WASM.

Reply:
then it has to change cmake file, which is annoying for users.

Reply:
I feel like we could detect `__wasm__` or similar in CMake. And it feels like the right tool, though maybe not the most trivial to implement.

Reply:
> I feel like we could detect `__wasm__` or similar in CMake. And it feels like the right tool, though maybe not the most trivial to implement.

Tbh i would always avoid wrapping around cmake, if platform macro works out.